### PR TITLE
after tests

### PR DESCRIPTION
**Pull request by Dorosh90**
- [DEMO LINK](https://Dorosh90.github.io/layout_moyo-header/)
- - [TEST REPORT LINK](https://Dorosh90.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,9 +6,85 @@
...
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet"href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="conteiner">
```
Comment (sTorba24) [inline src/index.html:15]:
```suggestion
    <header class="container">
```

File: src/index.html [hunk 2]
```diff
@@ -6,9 +6,85 @@
...
+      <a
+        class="logo"
+        href="http://localhost:61118/">
+        <img
+            src="./images/logo.png"
+            alt="logo"
+            href="#logo">
```
Comment (sTorba24) [inline src/index.html:22]:
```suggestion
        <img
           src="./images/logo.png"
           alt="logo"
           href="#logo"
         >
```

Comment (sTorba24) [reply src/index.html:22]: close opening tag on new line under tag start
attributes should have 2-space indentation

Fix everywhere please

**Review by sTorba24 — CHANGES_REQUESTED** [review]
Good work! Let's improve html :)

File: src/style.css [hunk 3]
```diff
@@ -1 +1,73 @@
-/* Styles go here */
+html {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  --color-action: #00acdc;
```
Comment (OleziO) [inline src/style.css:5]: Move to `:root`

**Review by OleziO — CHANGES_REQUESTED** [review]

**Review by sTorba24 — CHANGES_REQUESTED** [review]
Fix prev comments everywhere please, not for img only

File: src/style.css [hunk 4]
```diff
@@ -1 +1,73 @@
-/* Styles go here */
+:root {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
```
Comment (sTorba24) [inline src/style.css:4]: only color to :root

**Review by sTorba24 — CHANGES_REQUESTED** [review]
little fix need to do

**Review by sTorba24 — APPROVED** [review]
Approved :)

**Pull request closed without merging**